Makefile.mingw64.mk

```diff
@@ -135,6 +155,8 @@ build_zest:
 #
 
 TARGET_ZIP_FILE	:= $(BUILD_PATH)/zyn-fusion-windows-64bit-$(MODE).zip
```

Comment:
Looks like this is missing $(VER) for consistency sake.

Reply:
I added it to the zip name.